### DISCUSSION TITLE
Idea - park award progress bar

### DISCUSSION THREAD

**Discussion by ageekhere** [OpenRCT2/OpenRCT2#18977] [Feature requests]
The idea is to have a progress bar showing how far off the player is to getting a reward.
Also useful information could be added here in hover tips outlining what the player need to do to achieve or avoid a reward.
Maybe the higher the green bar the more chance of getting the reward in rewards which are given out by chance.

![img](https://user-images.githubusercontent.com/13173133/210034185-5a60747e-70d0-4402-8cf2-53c4e5fa3cbc.png)

**Comment by ocalhoun6** [comment]
I don't know...

Having the conditions of each award being somewhat of a mystery is part of what can make getting awards fun. Makes it easier to pretend that some imaginary organization is sending out representatives to various parks and then deciding on awards to give them.

-----

There's also a game balance issue. Since this would tell you which award you're up for this month, you could then specifically tailor things to win that award. (Or avoid that award if it's a negative one.)

For example, if you see that your park is being considered for 'best value park', you could easily reduce the entry price for just one month, ensuring that you win the award ... then raise it right back up afterward. Heck, timing it perfectly, you could probably just reduce the entry price on the 30th of the month, then raise it right back up on the 1st of the next month, after getting the award. So instead of actually having a good value, you'd only need to cheat the system for a few days.

Or, maybe your park is being considered for the 'worst park food' negative award. If you see that coming, you can quickly build more food stalls and avoid getting that award.

-----

Maybe this would make a good plugin, though?

**Reply by Crazycolbster** [reply]
This sounds like an awesome idea for a plugin! Now we just need an API that allows you to view things related to rewards. I think I'm going to open a feature request for exactly that

**Comment by J-PIE-314** [comment]
Sound like you should request this feature for this awards plugin: https://github.com/peterwooley/openrct2-award-eligibility
